### PR TITLE
Restructure ran manifest and allow profile per worker node

### PR DESCRIPTION
**Pull request by serngawy**
Signed-off-by: melserngawy <melserng@redhat.com>

### CODE REVIEW

File: manifests/sites/sample-site-du/06_sriov_network_node_policy_placementrule.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: apps.open-cluster-management.io/v1
 kind: PlacementRule
 metadata:
   name: placement-sample-site-du-sriov-nnode-policy
-  namespace: sample-site-du-subscription
+  namespace: sites-ran-subscriptions
```
Comment (yrobla) [inline manifests/sites/sample-site-du/06_sriov_network_node_policy_placementrule.yaml:5]: have you tried that with multiple sites? i wonder if we need one namespace per site, and if we will have problems when we add 2 subscriptions for 2 sites with the same namespace

Comment (serngawy) [reply manifests/sites/sample-site-du/06_sriov_network_node_policy_placementrule.yaml:5]: yes, I test it with different clusters profile combination and it works

File: README.md [hunk 2]
```diff
@@ -6,12 +6,12 @@ The purpose of this repository is to contain all the RAN manifests that is used
...
-    - Profile (which profile you want to apply to your cluster. It is either cu or du)
-        - Note: the cluste profile should match the cluster profile you used with ACM manifest.
+    - Profile (which profile you want to include in your cluster: cu, du, cu-du )
 
 - After executing the site_gen.sh script you will find a directory created under the manifest/sites/{with_YOUR_CLUSTER_NAME}.
 
-- You need to modify the complianceType/spec section in the 04_sriov_network_node_policy.yaml file under manifest/sites/{with_YOUR_CLUSTER_NAME} with the intf name and type that match your cluster nodes.
+- The CU and DU worker nodes will be labeled as [worker-cu, worker-du] based on worker node name prefix [cnfcu, cnfdu] defined at 01_node_autolabeler_policy.yaml
+
+- You need to modify 04_sriov_network_node_policy.yaml AND 07_sriov_network_policy.yaml under your new sites directory to match you worker nodes configurations.
```
Comment (yrobla) [inline README.md]: you/your

File: manifests/profile/du/06_performance_profile_policy.yaml [hunk 3]
```diff
@@ -48,4 +48,4 @@ spec:
                   numa:
                     topologyPolicy: "restricted"
                   realTimeKernel:
-                    enabled: true
+                    enabled: false
```
Comment (yrobla) [inline manifests/profile/du/06_performance_profile_policy.yaml]: why realtime kernel is disabled?

File: manifests/sites/cnf-cu-du-site/01_node_autolabeler_policy.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: v1
+                kind: ConfigMap
+                metadata:
+                  name: autorules
+                  namespace: node-autolabeler
+                data:
+                  rules1.properties: |
+                    name: cnfcu.* # As example to distingush between Nodes cnfcu ==> for CU and cnfdu ==> DU
```
Comment (yrobla) [inline manifests/sites/cnf-cu-du-site/01_node_autolabeler_policy.yaml]: distinguish

**Review by yrobla — CHANGES_REQUESTED** [review]
small typos/comments

File: manifests/profile/du/06_performance_profile_policy.yaml (hunk 3 above)
Comment (serngawy) [reply manifests/profile/du/06_performance_profile_policy.yaml]: just forget to re-enable

**Review by yrobla — APPROVED** [review]

**Pull request merged**